fulp_modules/antag_tips/html/abductor.html

```diff
@@ -0,0 +1,15 @@
+<html>
+	<center>
+	<h1>You are an Abductor!</h1>
+	<img src="abductor.png"/>
+	<p>You and your teammate have been chosen by the mothership to go and capture some humans.</p>
+	<p>There's two of you: the <b>scientist</b>, who must oversee the camera and surgery, and the <b>agent</b>, who must go down on the station to stun and cuff people.</p>
+	<p><img src="scitool.png"/>The scientist must utilize his <b>console</b> and <b>science tool</b>, to scan disguises and scan the agent for retrieval if neccesary.</p>
+	<p>The scientist must look for targets and send down the agent, after the target is captured the scientist must go down and use the science tool to mark as the teleport target.</p>
+	<p><img src="alienorgan.png"/>After that, he must do the <b>experimentation surgery</b>.</p>
```

Comment:
extraterrestrial experimental surgery. we dont want them doing default since it requires taking clothes off.